email_account_validity/account_validity.py

```diff
@@ -54,8 +84,8 @@ def parse_config(config: dict):
                 "'renew_at' is required when using email account validity"
             )
 
-        config["period"] = Config.parse_duration(config.get("period") or 0)
-        config["renew_at"] = Config.parse_duration(config.get("renew_at") or 0)
+        config["period"] = parse_duration(config["period"])
+        config["renew_at"] = parse_duration(config["renew_at"])
         return config
```

Comment:
it'd be great to use an `attrs` rather than a dict here. maybe one for the future.